_posts/reflect-92.md

```diff
@@ -0,0 +1,202 @@
+---
```

Comment:
Please always name the files with the same pattern as the other files. The date is not important in a draft, it just has to be a valid date. Otherwise the page won't render. I did this already, now.

If you do that, and add a date in the header (see comment below). You can preview your article once you did a PR (there is a preview link in the PR after a couple minutes). This would really help because then we could avoid some of the issues like bad image links and code snippets that need a scrollbar in the first place.